spec/requests/as_content_owner/content_owner_spec.rb

```diff
@@ -53,19 +53,15 @@
     describe '.includes(:claim)' do
       let(:videos) { $content_owner.videos.includes(:claim) }
       let(:video_with_claim) { videos.find{|v| v.claim.present?} }
+      let(:video_with_claim_with_asset) do
+        videos.find{|v| v.try(:claim).try(:asset)}
+      end
 
       specify 'eager-loads the claim of each video' do
```

Comment:
and change this to

``ruby`
specify 'eager-loads the claim of each video and its asset' do
  expect(video_with_claim.claim).to be_a Yt::Claim
  expect(video_with_claim.claim.id).to be_a String
  expect(video_with_claim.claim.video_id).to eq video_with_claim.id

  expect(video_with_claim.claim.asset).to be_a Yt::Asset
  expect(video_with_claim.claim.asset.id).to be_a String 
```